Fix undefined reference to  IconLoader::*

Fixes:
 obj/chrome/browser/libbrowser.a(obj/chrome/browser/browser/icon_loader.o):icon_loader.cc:function IconLoader::ReadGroup(): error: undefined reference to 'IconLoader::GroupForFilepath(base::FilePath const&)'
 obj/chrome/browser/libbrowser.a(obj/chrome/browser/browser/icon_loader.o):icon_loader.cc:function IconLoader::ReadGroup(): error: undefined reference to 'IconLoader::GetReadIconTaskRunner()'
 obj/chrome/browser/libbrowser.a(obj/chrome/browser/browser/icon_loader.o):icon_loader.cc:function IconLoader::ReadGroup(): error: undefined reference to 'IconLoader::ReadIcon()'

Signed-off-by: Damian Wrobel <dwrobel@ertelnet.rybnik.pl>
---

Index: chromium-76.0.3809.132/chrome/browser/icon_loader.cc
===================================================================
--- chromium-76.0.3809.132.orig/chrome/browser/icon_loader.cc
+++ chromium-76.0.3809.132/chrome/browser/icon_loader.cc
@@ -3,47 +3,22 @@
 // found in the LICENSE file.
 
 #include "chrome/browser/icon_loader.h"
-
-#include <utility>
-
-#include "base/bind.h"
-#include "base/task/post_task.h"
-#include "base/task/task_traits.h"
-#include "base/threading/thread_task_runner_handle.h"
-#include "content/public/browser/browser_thread.h"
-
-using content::BrowserThread;
+#include "base/logging.h"
 
 // static
-IconLoader* IconLoader::Create(const base::FilePath& file_path,
-                               IconSize size,
-                               IconLoadedCallback callback) {
-  return new IconLoader(file_path, size, std::move(callback));
+IconLoader::IconGroup IconLoader::GroupForFilepath(
+    const base::FilePath& file_path) {
+  NOTIMPLEMENTED();
+  return IconLoader::IconGroup();
 }
 
-void IconLoader::Start() {
-  target_task_runner_ = base::ThreadTaskRunnerHandle::Get();
-
-  base::PostTaskWithTraits(
-      FROM_HERE, traits(),
-      base::BindOnce(&IconLoader::ReadGroup, base::Unretained(this)));
-}
-
-IconLoader::IconLoader(const base::FilePath& file_path,
-                       IconSize size,
-                       IconLoadedCallback callback)
-    : file_path_(file_path),
-#if !defined(OS_ANDROID)
-      icon_size_(size),
-#endif  // defined(OS_ANDROID)
-      callback_(std::move(callback)) {
+// static
+scoped_refptr<base::TaskRunner> IconLoader::GetReadIconTaskRunner() {
+  NOTIMPLEMENTED();
+  return nullptr;
 }
 
-IconLoader::~IconLoader() {}
-
-void IconLoader::ReadGroup() {
-  group_ = GroupForFilepath(file_path_);
-
-  GetReadIconTaskRunner()->PostTask(
-      FROM_HERE, base::BindOnce(&IconLoader::ReadIcon, base::Unretained(this)));
+void IconLoader::ReadIcon() {
+  NOTIMPLEMENTED();
+  delete this;
 }
